app/graphql/types/query.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class Query < GraphQL::Schema::Object
+    graphql_name 'Query'
+
+    field :model, Types::Model,
```

Comment:
When not logged in, I am getting:
```
{
  "data": null,
  "errors": [
    {
      "message": "Cannot return null for non-nullable field Query.model"
    }
  ]
}
```

Is that intentional? Or should this field have `null: true`?

Reply:
Good catch, I have added `null: true`. Same happens when you query for a model that does not exist.